src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible"   content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+
+  <body>
+  Hello World!
+  </body>
```

Comment:
```suggestion
  <body>
    Hello World!
  </body>
```
Pay attention to formatting. There should be indentations here
You may read about it here https://mate-academy.github.io/style-guides/htmlcss.html